doc/connecting.md

```diff
@@ -20,26 +20,54 @@ myst:
 
 # Connecting to a database
 
-Learn how to connect to various databases using JupySQL.
+JupySQL offers several ways to configure a database connection. In this guide, we discuss the pros and cons of each.
+
+## Connecting with a `.ini` file
+
+```{versionchanged} 0.10.0
+```
+
+Using a `.ini` file is the recommended way to connect to databases. By default, JupySQL reads the `~/.jupysql/connections.ini` file, but you can change this setting. A `.ini` file looks like this:
```

Comment:
Shouldn't it be looking in the path where `jupysql` folder actually exists? e.g., in my machine the path doesn't exist:
```
(jupysql) neelashasen@192 jupysql % ls /Users/neelashasen/.jupysql
ls: /Users/neelashasen/.jupysql: No such file or directory
```

<img width="732" alt="Screenshot 2023-08-20 at 12 07 14 PM" src="https://github.com/ploomber/jupysql/assets/29096135/fb82d644-b100-4bbb-98e0-16a735e01021"> 

I went on to create a `connections.ini` file in my jupysql directory:
<img width="367" alt="Screenshot 2023-08-20 at 12 03 30 PM" src="https://github.com/ploomber/jupysql/assets/29096135/79283a91-d3f4-48e4-92fa-082081fb8f3c">

And it throws error:

<img width="1099" alt="Screenshot 2023-08-20 at 12 04 05 PM" src="https://github.com/ploomber/jupysql/assets/29096135/d48daa4e-d68e-4213-a099-f103c624f35a">


Maybe there should be some warnings / errors if `~/.jupysql/connections.ini` is not a valid path?

Reply:
the path doesn't exist, the user has to create it. I don't think we should show warnings because this feature is optional so warning them that the .ini file doesn't exist will be confusing.

I don't understand the error you're reporting. It's likely failing because the file isn't there, this is how it looks on my end:

```
(jupysql) eduardo@macbookair dev/jupysql (dev/default-conn) » cat ~/.jupysql/connections.ini
[default]
database = :memory:
drivername = duckdb

[disk]
drivername = duckdb
database = /Users/eduardo/.jupysql/duck.db

(jupysql) eduardo@macbookair dev/jupysql (dev/default-conn) » ipython
Python 3.10.12 (main, Jul  5 2023, 15:02:25) [Clang 14.0.6 ]
Type 'copyright', 'credits' or 'license' for more information
IPython 8.14.0 -- An enhanced Interactive Python. Type '?' for help.

In [1]: %load_ext sql
Found pyproject.toml from '/Users/eduardo/dev/jupysql'

In [2]: %sql --connections
Out[2]:
Active connections:
+---------+--------------------+---------+
| current |        url         |  alias  |
+---------+--------------------+---------+
|    *    | duckdb:///:memory: | default |
+---------+--------------------+---------+

In [3]:
```